kumo-api/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>kumo</artifactId>
+        <groupId>com.kennycason</groupId>
+        <version>1.9-SNAPSHOT</version>
```

Comment:
remove the snapshot version, just bump to 1.9. Though your last PR is at 1.9 already. but don't worry about it, i can bump afterwords.